doc/manual/src/advanced-topics/distributed-builds.md

```diff
@@ -36,16 +36,9 @@ error: cannot connect to 'mac'
 then you need to ensure that the `PATH` of non-interactive login shells
 contains Nix.
 
-> **Warning**
->
-> If you are building via the Nix daemon, it is the Nix daemon user account (that is, `root`) that should have SSH access to a user (not necessarily `root`) on the remote machine.
->
-> If you can’t or don’t want to configure `root` to be able to access the remote machine, you can use a private Nix store instead by passing e.g. `--store ~/my-nix` when running a Nix command from the local machine.
-
-The list of remote machines can be specified on the command line or in
-the Nix configuration file. The former is convenient for testing. For
-example, the following command allows you to build a derivation for
-`x86_64-darwin` on a Linux machine:
+The [list of remote machines](@docroot@/command-ref/conf-file.md#conf-builders) can be specified on the command line or in the Nix configuration file.
+The former is convenient for testing.
```

Comment:
```suggestion
The former is convenient for testing.
```
It's also useful for separating stuff, for example I have access to some community builders which are nice for building nixpkgs PRs but I don't want to do my personal builds on them because I don't want to waste resources and it would be bad security practices to do so since so many people have trusted access and could in theory manipulate builds.

Reply:
```suggestion
```

Probably shouldn't even be there. It's from the original guide-style document, and reference documentation should not judge what stuff is good for or not unless it's factual and unambiguous. This is one of these "you can" situations. Sure you can, but you can do a lot of things and this is not the place to cover them all.